spec/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'user'
+
+describe User do
+  describe '.valid' do
+    it 'checks password and username is correct' do
```

Comment:
might be slightly neater to use context here. So context 'Password and username are correct' then under that it "returns true". context 'Password and username are inccorrect', under that it 'returns false'

Reply:
good advice, I need to get more used to using context 😅